tests/utils/exec/rosacli/cmd_runner.go

```diff
@@ -186,7 +186,7 @@ func (r *runner) Run() (bytes.Buffer, error) {
 
 		err = cmd.Run()
 
-		log.Logger.Debugf("Get Combining Stdout and Stder is :\n%s", output.String())
+		log.Logger.Infof("Get Combining Stdout and Stder is :\n%s", output.String())
```

Comment:
do we really want that by default in logs ?

Reply:
Yes, otherwise we cannot debug what happened.